npm_and_yarn/helpers/test/pnpm/lockfile-parser.test.js

```diff
@@ -0,0 +1,61 @@
+const {
+    parseLockfile,
+} = require("../../lib/pnpm");
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+
+describe("parseLockfile", () => {
+
+    let tempDir;
+    beforeEach(() => {
+        tempDir = fs.mkdtempSync(os.tmpdir() + path.sep);
+    });
+    afterEach(() => fs.rm(tempDir, { recursive: true }, () => {}));
+
+    function copyDependencies(sourceDir, destDir) {
+        const srcPnpmYaml = path.join(
+            __dirname,
+            `fixtures/parser/${sourceDir}/pnpm-lock.yaml`
+        );
+        fs.copyFileSync(srcPnpmYaml, `${destDir}/pnpm-lock.yaml`);
+    }
+
+    it("no lock file change", async () =>{
+        copyDependencies("no_lockfile_change", tempDir);
+        const result = await parseLockfile(tempDir);
+
+        expect(result.length).toEqual(398);
+    })
+
+    it("only dev dependency", async () =>{
```

Comment:
Nitpick, applies throughout this file: if we're using `describe`/`it` syntax, can we use test descriptions that follow the `it + expected behavior` format? I'm not 100% sure what this test is testing.

Here are a couple examples from elsewhere in the test suite:

https://github.com/dependabot/dependabot-core/blob/1bd3b5d49e37c03a2448444dba777e41bea8e6bf/npm_and_yarn/helpers/test/yarn/updater.test.js#L28

https://github.com/dependabot/dependabot-core/blob/1bd3b5d49e37c03a2448444dba777e41bea8e6bf/npm_and_yarn/helpers/test/yarn/conflicting-dependency-parser.test.js#L30

Reply:
@bdragon Thanks and updated.

Reply:
Sorry if my comment wasn't clear. I was just checking to see if this was something we needed to handle, but what you had was fine so there was no need to change it.